.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -61,7 +61,7 @@ body:
   attributes:
     label: Config Connector Version
     description: | 
-      kubectl get ns cnrm-system -o jsonpath='{.metadata.annotations.cnrm\\.cloud\\.google\\.com/version}'
+      kubectl get ns cnrm-system -o jsonpath='{.metadata.annotations.cnrm\\.cloud\\.google\\.com/version}{"\n"}'
```

Comment:
I think this has to be \\n given the previous escaping.

Reply:
Good point. Done.